envsec/internal/jetcloud/git.go

```diff
@@ -0,0 +1,31 @@
+package jetcloud
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func createGitIgnore(wd string) error {
+	gitIgnorePath := filepath.Join(wd, dirName, ".gitignore")
+	return os.WriteFile(gitIgnorePath, []byte("*"), 0600)
+}
+
+func gitRepoURL(wd string) (string, error) {
+	cmd := exec.Command("git", "config", "--get", "remote.origin.url")
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", err
+	}
+	return strings.TrimSpace(string(output)), nil
+}
+
+func gitSubdirectory(wd string) (string, error) {
+	cmd := exec.Command("git", "rev-parse", "--show-prefix")
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", err
```

Comment:
Same thing, what happens if the user does not have `git` installed?

Reply:
error is ignored, we se empty string